### PR TITLE
Resume only

### PR DESCRIPTION
**Pull request by D4nte**
New command `nectar wrap-up` that finishes executing ongoing swaps.
- It does not publish new swaps
- It rejects any new take request
- Exists once all swaps are finished

UX would be:
```
nectar trade
# CTRL-C
nectar resume-only
```

Patches can be reviewed individually.

### CODE REVIEW

File: src/command/trade.rs [hunk 1]
```diff
@@ -352,37 +343,13 @@ fn handle_network_event(
     ethereum_wallet: Arc<ethereum::Wallet>,
     bitcoin_connector: Arc<comit::btsieve::bitcoin::BitcoindConnector>,
     ethereum_connector: Arc<comit::btsieve::ethereum::Web3Connector>,
+    take_request_handler: F,
```
Comment (bonomat) [inline src/command/trade.rs:346]: Why do you pass in the logic for processing the take request as a function from the outside?

**Comment by bonomat** [comment]
Thanks for asking me for a review. I don't know anything about the codebase so will leave the big comments to others :) 

I've added one question out of interest though.

File: src/command/trade.rs (hunk 1 above)
Comment (D4nte) [reply src/command/trade.rs]: Because depending on whether I am in `trade` or `wrap-up` then I would process the take request differently.

**Comment by D4nte** [comment]
> Thanks for asking me for a review. I don't know anything about the codebase so will leave the big comments to others :)
> 
> I've added one question out of interest though.

I'd like some UX input from you and @thomaseizinger 😉

**Comment by bonomat** [comment]
> > Thanks for asking me for a review. I don't know anything about the codebase so will leave the big comments to others :)
> > I've added one question out of interest though.
> 
> I'd like some UX input from you and @thomaseizinger 😉

```bash
nectar trade
# CTRL-C
nectar wrap-up
```

I think that would do the job 👍 
Maybe printing on the console how many swaps are pending would be a plus.

File: src/command/resume_only.rs [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            Arc::clone(&db),
+            Arc::clone(&bitcoin_wallet),
+            Arc::clone(&ethereum_wallet),
+            Arc::clone(&bitcoin_connector),
+            Arc::clone(&ethereum_connector),
+            swap,
+        )
+        .await?;
+        handle_finished_swap(finished_swap, Arc::clone(&db), Arc::clone(&history))
+    }
```
Comment (luckysori) [inline src/command/resume_only.rs:65]: :mega: I think we should be joining all these futures rather than awaiting one by one.

Comment (D4nte) [reply src/command/resume_only.rs]: Done.

**Pull request merged**